internal/device/device.go

```diff
@@ -0,0 +1,53 @@
+package device
+
+import (
+	"fmt"
+	"time"
+)
+
+func AmICharged() {
+	fmt.Println("Am I charged???")
+	fmt.Println()
+	powerStatus := GetPower()
+
+	percent, status := PowerStats(powerStatus)
+
+	if isCharged(status) {
+		if is100Percent(percent) {
+			go CompletedNotifyAndRerun()
```

Comment:
I'm not absolutely clear why this runs in a separate goroutine; what's your thinking here?

Reply:
The idea was to exit the current function call immediately.  Now when I think about it,  I feel its not necessary here.

Reply:
There's a danger that the binary might exit before some of these goroutines are complete, isn't there?